subcommands/git-sync/test-completion.sh

```
@@ -0,0 +1,83 @@
+test_completion() {
+    local input="$1"
+    echo -n "Completing '$input': "
+    
+    # Parse into words
+    read -r -a COMP_WORDS <<< "$input"
+    COMP_CWORD=$((${#COMP_WORDS[@]} - 1))
+    COMP_LINE="$input"
+    COMP_POINT=${#COMP_LINE}
+    
+    # Set cur to last word
+    cur="${COMP_WORDS[COMP_CWORD]}"
```

Shellcheck (0.10.0)

[warning] cur appears unused. Verify use (or export if used externally).

(SC2034)